.drone.yml

```diff
@@ -3457,6 +3457,17 @@ steps:
   - trivy --exit-code 1 --severity HIGH,CRITICAL grafana/grafana:latest-ubuntu
   - trivy --exit-code 1 --severity HIGH,CRITICAL grafana/grafana:main-ubuntu
 
+- name: slack
+  image: plugins/slack
+  settings:
+    channel: grafana-backend
+    template: Nightly docker image scan job for {{repo.name}} failed
```

Comment:
Maybe we can add some more info - links to drone failed pipeline page?

(check template references [here](http://plugins.drone.io/drone-plugins/drone-slack/))

Reply:
(another source [here](https://github.com/grafana/grafana/blob/main/scripts/lib.star#L76))